backend/src/main/java/gov/cdc/usds/simplereport/service/TestResultUploadService.java

```diff
@@ -172,16 +167,90 @@ public UploadResponse getUploadSubmission(UUID id)
             .findByInternalIdAndOrganization(id, org)
             .orElseThrow(InvalidBulkTestResultUploadException::new);
 
+    var r = getRSAuthToken();
```

Comment:
inline this please since its only used once.

Reply:
ah, I think this was out-of-line for the sake of local debugging and I forgot to revert it back - good catch